frame/benchmarking/src/lib.rs

```diff
@@ -165,18 +171,29 @@ macro_rules! impl_benchmark {
 					let steps = steps.get(idx).unwrap_or(&prev_steps);
 					prev_steps = steps;
 
+					let min = mins.get(idx).unwrap_or(&low);
```

Comment:
`let min = mins.get(idx).cloned().unwrap_or(low);` looks more intuitive to me, then you can remove all `*` later on